errata_tool/erratum.py

```diff
@@ -557,7 +557,7 @@ def setState(self, state):
                                   state.upper())
 
     def _addBug(self, b):
-        if type(b) is not int:
+        if not isinstance(b, int):
```

Comment:
This linting cleanup could be split out and merged by inspection, as it is not directly related to this pull request.

Reply:
not a problem here